rustup/sync.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu
+[[ $DEBUG = true ]] && set -x
+
+echo "Running script"
+
+export UPSTREAM="${UPSTREAM:-https://static.rust-lang.org/}"
+export GC="${GC:-1}"
+export TARGETS="${TARGETS:-x86_64-unknown-linux-gnu}"
+export URL="${URL:-http://127.0.0.1:8000.}"
+
+rustup-mirror --upstream-url "$UPSTREAM" \
```

Comment:
```suggestion
exec rustup-mirror --upstream-url "$UPSTREAM" \
```

参考 https://github.com/ustclug/ustcmirror-images/blob/master/CONTRIBUTING.md#%E7%BA%A6%E5%AE%9A